rust_src/src/data.rs

```diff
@@ -0,0 +1,51 @@
+use remacs_macros::lisp_fn;
```

Comment:
"//! Primitive operations on Lisp data types for GNU Emacs Lisp interpreter." is missing here.

Reply:
Good point!